src/components/home-container/index.jsx

```diff
@@ -40,26 +44,11 @@ function HomeContainer() {
             onDeleteClick={handleDeleteClick}
           />
         )}
-        {isTaskAvailable &&
-          tasks.map((task) => (
-            <TaskCard
-              key={task.id}
-              taskId={task.id}
-              title={task.title}
-              createdAt={task.createdAt}
-              isCompleted={task.isCompleted}
-              completedAt={task.completedAt}
-            />
-          ))}
+
+        <ExistingTaskCardContaienr tasks={tasks.slice(0, numOfCardVisible)} />
```

Comment:
```suggestion
        <TaskCardContaienr tasks={tasks.slice(0, numOfCardVisible)} />
```

isn't that more simple?